### PR TITLE
Univers Simu Mods Added v2

### PR DESCRIPTION
**Pull request by Lactic68**
Warning to @ddewar :
Don't know how to apply UA only if included bale autoload is not the active configuration.
If you don't mind, I will better leave to your hands the modification about this.

Everything else should be fixed. Thank you so much for all your feedback and advices.

### CODE REVIEW

File: config/SupportedVehicles.xml [hunk 1]
```diff
@@ -96,6 +96,33 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
+            <loadingArea offset="0.018 1.39 1.352" width="2.195" height="4.48" length="7.78"/>
```
Comment (ddewar) [inline config/SupportedVehicles.xml:101]: Looks pretty good, I think we should reduce the height a little though:

image

File: config/SupportedVehicles.xml [hunk 2]
```diff
@@ -96,6 +96,33 @@
...
+			<options noLoadingIfUnfolded="true"/>
```
Comment (ddewar) [inline config/SupportedVehicles.xml:102]: This option worked perfectly. Thanks for adding it.

File: config/SupportedVehicles.xml [hunk 3]
```diff
@@ -96,6 +96,33 @@
...
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_920/rpx920.xml">
```
Comment (ddewar) [inline config/SupportedVehicles.xml:104]: Same as above with height, otherwise great.

File: config/SupportedVehicles.xml [hunk 4]
```diff
@@ -96,6 +96,33 @@
...
+            <loadingArea offset="0.095 1.39 0.816" width="2.195" height="4.48" length="8.662"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_1226/rpx1226.xml">
```
Comment (ddewar) [inline config/SupportedVehicles.xml:108]: Same as above with height, otherwise great.

File: config/SupportedVehicles.xml [hunk 5]
```diff
@@ -96,6 +96,33 @@
...
+            <loadingArea offset="0.059 1.39 -0.657" width="2.195" height="4.48" length="11.831"/>
+			<options noLoadingIfUnfolded="true"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPEX_7519/SPEX.xml">
+            <loadingArea offset="-0.02 0.978 -2.661" width="1.991" height="2.642" length="5.268"/>
```
Comment (ddewar) [inline config/SupportedVehicles.xml:113]: Loading area could use a little more length if we wanted. Very optional though:

image

You can see above the loading zone comes a bit short of where the ramp starts.

Comment (ddewar) [reply config/SupportedVehicles.xml:113]: Same with width, there's probably room for a little more if we want:

image

File: config/SupportedVehicles.xml [hunk 6]
```diff
@@ -96,6 +96,33 @@
...
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSEL10006/LSEL10006_Square.xml">
+            <loadingArea offset="0.015 1.137 -0.161" width="2.352" height="5.301" length="9.627"/>
```
Comment (ddewar) [inline config/SupportedVehicles.xml:116]: Same height issues:

image

Comment (ddewar) [reply config/SupportedVehicles.xml:116]: I think we want to move the offset to: ```offset="0 1.137 -0.161"``` It's pretty rare we need to move the offset in the x-direction. You can see here, it slightly has moved it under one side of the edge of the trailer on the left:

image

File: config/SupportedVehicles.xml [hunk 7]
```diff
@@ -96,6 +96,33 @@
...
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSP9004/LSP9004_Square.xml">
```
Comment (ddewar) [inline config/SupportedVehicles.xml:118]: Same as above. Height and offset.

File: config/SupportedVehicles.xml [hunk 8]
```diff
@@ -96,6 +96,33 @@
...
+            <loadingArea offset="0.031 1.137 -0.654" width="2.352" height="5.042" length="8.432"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PerardPack/platosol/platosol.xml">
+            <loadingArea offset="-0.008 0.923 -0.127" width="2.109" height="3.676" length="5.651"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PerardPack/rpe26/rpe26.xml">
```
Comment (ddewar) [inline config/SupportedVehicles.xml:124]: This one looks just about perfect. Only suggestion I have is add option: ```<options enableRearLoading="true"/>``` if you want to allow rear autoloading via pallet forks.

File: config/SupportedVehicles.xml (hunk 5 above)
Comment (ddewar) [reply config/SupportedVehicles.xml:113]: Suggestion add ```<options enableRearLoading="true"/>``` if you want to allow rear autoloading via pallet forks.

File: config/SupportedVehicles.xml [hunk 9]
```diff
@@ -96,6 +96,33 @@
...
+        <vehicleConfiguration configFileName="FS22_PongePack/SPEX_7519/SPEX.xml">
+            <loadingArea offset="-0.02 0.978 -2.661" width="1.991" height="2.642" length="5.268"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSEL10006/LSEL10006_Square.xml">
+            <loadingArea offset="0.015 1.137 -0.161" width="2.352" height="5.301" length="9.627"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_RollandPack/LSP9004/LSP9004_Square.xml">
+            <loadingArea offset="0.031 1.137 -0.654" width="2.352" height="5.042" length="8.432"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PerardPack/platosol/platosol.xml">
```
Comment (ddewar) [inline config/SupportedVehicles.xml:121]: Same as above. Height and offset.

**Review by ddewar — COMMENTED** [review]
Overall, this looks really good. It was all perfectly functional, and I did not see any errors in the log.  Thank you for adding to the readme and mod desc. It will make it much easier for me to keep track of!

Some suggestions:

1. Check the heights on some of your loading zones. Not that it will cause too many issues, but most of the trailers become really top heavy. My suggestion is keeping the heights around 3 or less. 
2. It looks like some of your offsets have an 'x' value. This is slightly shifting the zones off-center on the trailers. I would suggest moving these all back to 0. 
3. I will handle the selectedConfigs option if we need it. It looks like as of now we don't as the autoload changes the trailer and makes UA not work anyway.

Feel free to reach out with any questions or anything you don't feel like doing. As these are your configs you are adding, I am happy to let you decide what truly needs to be included or not. If you do not want to change anything else let me know and I will get these merged in!

**Comment by ddewar** [comment]
> > Overall, this looks really good. It was all perfectly functional, and I did not see any errors in the log. Thank you for adding to the readme and mod desc. It will make it much easier for me to keep track of!
> > Some suggestions:
> > 
> > 1. Check the heights on some of your loading zones. Not that it will cause too many issues, but most of the trailers become really top heavy. My suggestion is keeping the heights around 3 or less.
> 
> Hi @ddewar , For some player it is a nice feature to be able to stack rows of pallet. If we reduce the heights to 3, will this be still possible ?

Yes! I am pretty sure 3 usually allows stacking about 3 or 4 pallets heigh depending on the pallet. With some of the trailers in the PR the pallet stacks were getting up to 9 or so pallets heigh.

**Comment by Lactic68** [comment]
Thank you very much for all your comments and additions.
Closing this PR in order to submit another one

**Pull request closed without merging**